### DISCUSSION TITLE
How to convert numeric character to Unicode string?

### DISCUSSION THREAD

**Discussion by johannes-collab** [Z3Prover/z3#5359] [Q&A] [answered]
With the old sequence solver, we used `seq.unit` to convert characters to strings.

```smtlib
(declare-const str String)
(assert (= str (seq.unit #x61)))
```

This doesn't seem to be possible with the Unicode string solver (the new default). Am I missing something here?

**Comment by krobelus** [comment, marked as answer]
Nevermind, it's `str.from_code`, as per http://smtlib.cs.uiowa.edu/theories-UnicodeStrings.shtml
